app/src/components/domain/RiskImminentEvents/Pdc/EventDetails/index.tsx

```diff
@@ -18,6 +45,9 @@ interface Props {
     data: PdcEventItem;
     exposure: PdcExposure | undefined;
     pending: boolean;
+    onLayerChange: (value: boolean, name: LayerType) => void;
```

Comment:
What does onLayerChange mean? What are we changing on the layer?

Reply:
this is for interactive layer, as to hide and unhide layers on map

Reply:
Let's change the prop names accordingly